gravitee-apim-console-webui/src/management/api/proxy/properties/properties-import-dialog/properties-import-dialog.component.html

```diff
@@ -0,0 +1,70 @@
+<!--
+
+    Copyright (C) 2015 The Gravitee team (http://gravitee.io)
+    
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+    
+            http://www.apache.org/licenses/LICENSE-2.0
+    
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+-->
+<div class="title">
+  <div>
+    <h2 mat-dialog-title>Import properties</h2>
+    <span>
+      Easily import your API property list. If the properties already exist, their values will be overwritten. If the properties already
+      exist and are encrypted, the import will be skipped.
+    </span>
+  </div>
+  <mat-icon class="title__cancel"></mat-icon>
+  <button mat-icon-button [attr.aria-label]="'Close'" [mat-dialog-close]="">
+    <mat-icon class="title__cancel" svgIcon="gio:cancel"> </mat-icon>
+  </button>
+</div>
+<mat-dialog-content [formGroup]="formGroup">
+  <mat-form-field class="field">
+    <textarea matInput formControlName="properties" placeholder='NEW_PROPERTY_KEY"New property value"' rows="10"></textarea>
```

Comment:
Is this placeholder correct?
Should we have a space between key and new `placeholder='NEW_PROPERTY_KEY "New property value"'`

Reply:
ha right. not a space but an `=`

`NEW_PROPERTY_KEY="New property value"`